repository/s.json

```diff
@@ -3523,6 +3523,21 @@
 				}
 			]
 		},
+		{
+			"name": "SSH-Panel",
+			"details": "https://github.com/Haiquan-27/SSH-Panel",
+			"author": "Haiquan",
+			"issues": "https://github.com/Haiquan-27/SSH-Panel/issues",
+			"readme": "https://raw.githubusercontent.com/Haiquan-27/SSH-Panel/main/README.md",
```

Comment:
You don't need "issues" or "readme" here since they're just the defaults for GitHub.

Reply:
i have deleted " issues" and "readme"